examples/ChatQnA/deploy/gaudi.md

```diff
@@ -813,16 +813,17 @@ data: b'</s>'
 data: [DONE]
 ```
 
-##### Gaurdrail Microservice
-If you had enabled Gaurdrail microservice, access via the below curl command
+##### Guardrail Microservice

```

Comment:
This should either be an H3 (with ###) if it's at the same level as the `### Megaservice` heading above it or an H4 it it's a subheading under that Megaservice heading.  You shouldn't skip from H3 to H5 as this currently is.